CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v0.53.0 (unreleased)
+
+### Breaking changes
+
+- Drop python 3.9 support per [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).
+
+### Features
+
+- Build wheels for [python 3.13](https://peps.python.org/pep-0719/). (These wheels are available on PyPI, but other pycontrails dependencies may not yet support python 3.13.)
+
+### Fixes
+
+- Fix `PycontrailsRegularGridInterpolator` for compatibility with the latest scipy version.
```

Comment:
I think it's related, but I didn't actually enable support for spline interpolation -- I just added a `_spline` attribute to avoid a runtime error. It's definitely a larger task to fully support what the scipy interpolator can do. It would be nice to make pycontrails interpolation more harmonious with scipy ... but IIRC, the scipy interpolator is still doing some casts from float32 to float64, and we should avoid this for weather data.